File: app/src/main/java/com/techradge/fabler/ui/story/StoryClickListener.java
package com.techradge.fabler.ui.story;

import com.techradge.fabler.data.model.Story;

public interface StoryClickListener {
    void onStoryClick(int position, Story story);

    void onBookmarked(int position, Story story);

    void onBookmarkRemoved(int position, Story story);
}

File: app/src/main/java/com/techradge/fabler/ui/story/StoryClickListener.kt
package com.techradge.fabler.ui.story

import com.techradge.fabler.data.model.Story

interface StoryClickListener {
    fun onStoryClick(position: Int, story: Story)

    fun onBookmarked(position: Int, story: Story)

    fun onBookmarkRemoved(position: Int, story: Story)
}